### PR TITLE
Added list of unnecessary KDE packages.

### PR DESCRIPTION
**Pull request by xshram**

### CODE REVIEW

File: source/tips-and-tricks.rst [hunk 1]
```diff
@@ -87,10 +87,14 @@
 
 .. code-block:: text
 
-    sudo dnf remove gtk2
+    sudo dnf remove GTK2
```
Comment (xvitaly) [inline source/tips-and-tricks.rst]: Do not change case. `gtk2` package use lower-case.

File: source/tips-and-tricks.rst [hunk 2]
```diff
@@ -87,10 +87,14 @@
...
+
+Удалим остальные KDE приложения:
+
+.. code-block:: text
+
+    sudo dnf remove  discover krdc akonadi qt abrt PackageKit kdepim-runtime-libs kde-connect kdeconnectd ark dragon kontact okular gwenview ktorrent kget konversation konqueror falkon kmail dnfdragora kdewallet krusader kwrite spectacle krfb akregator juk kamoso k3b calligra* kfind kgpg kmouth kmag
 
-.. index:: bug, missing library, libcurl-gnutls
-.. _libcurl-workaround:
```
Comment (xvitaly) [inline source/tips-and-tricks.rst]: Do not change indexes of another questions.

File: source/tips-and-tricks.rst [hunk 3]
```diff
@@ -87,10 +87,14 @@
...
 .. code-block:: text
 
-    sudo dnf remove gtk2
+    sudo dnf remove GTK2
+
+Удалим остальные KDE приложения:
+
+.. code-block:: text
+
+    sudo dnf remove  discover krdc akonadi qt abrt PackageKit kdepim-runtime-libs kde-connect kdeconnectd ark dragon kontact okular gwenview ktorrent kget konversation konqueror falkon kmail dnfdragora kdewallet krusader kwrite spectacle krfb akregator juk kamoso k3b calligra* kfind kgpg kmouth kmag
```
Comment (xvitaly) [inline source/tips-and-tricks.rst]: After removal of KWrite, Gweenview, Ark and Okular system will be unusable.

**Comment by xshram** [comment]
Your wishes satisfied.

**Pull request merged**